app/scripts/directives/datepicker.js

```diff
@@ -30,12 +30,9 @@ angular
       },
       link: function (scope, element) {
         var datePickerElement = angular.element(element).find('.datepicker');
-        var initialDate =
-          scope.localModel && scope.monthYearOnly
-            ? scope.localModel
-            : scope.localModel
-            ? moment(new Date(scope.localModel)).format('MM/DD/YYYY hh:mm A')
-            : null;
+        var initialDate = scope.localModel
+          ? moment(new Date(scope.localModel)).format('YYYY-MM-DD HH:mm:ss')
+          : null;
```

Comment:
If you're setting the variable `initialDate` back to what it was previously, below is what it was. The `A` just stands for AM/PM, but the inputs show it anyway, even when you remove it. (I haven't checked all inputs but most)

```suggestion
        var initialDate = scope.localModel
          ? moment(scope.localModel).format('YYYY-MM-DD HH:mm A')
          : null;
```

Reply:
Setting the initial date to a certain format is actually just to make sure the datepicker can read it. The datepicker itself sets the format. So each datepicker in different parts of the page will format the display of the date in the datepicker differently.